Kickstarter-iOS/Views/Controllers/ManagePledgeViewController.swift

```diff
@@ -156,6 +160,14 @@ final class ManagePledgeViewController: UIViewController {
       .observeValues { [weak self] project, backing in
         self?.goToCancelPledge(project: project, backing: backing)
       }
+
+    self.viewModel.outputs.showSuccessBannerWithMessage
+      .observeForUI()
```

Comment:
Sometimes we use `.observeForUI` other times we use `.observeForControllerAction()` is there any difference?

Reply:
My understanding on the difference between these two is that there was some bug in the ReactiveSwift scheduler before that required this distinction. I should actually did in to see whether that was since fixed.

For consistency though I'll update this.